gslib/commands/acl.py

```diff
@@ -339,6 +342,45 @@ class AclCommand(Command):
           'ch': _ch_help_text
       },
   )
+  CannedGcsAcl = ['private', 'public-read', 'project-private',
+                    'public-read-write', 'authenticated-read',
+                    'bucket-owner-read', 'bucket-owner-full-control']
+  def get_gcloud_storage_args(self):
+    if self.args[0] == 'get':
+      url = self.args[1]
+      object_or_bucket = StorageUrlFromString(url)
+      if object_or_bucket.IsObject():
+        command_group = 'objects'
+      else:
+        command_group = 'buckets'
+      gcloud_storage_map = GcloudStorageMap(gcloud_command=[
+            'alpha', 'storage', command_group, 'describe', url, '--format=json[acl]'
```

Comment:
Try testing this command in the terminal to see if the shim output matches the gsutil output without the shim

Reply:
Here is the output when I try both commands. https://docs.google.com/document/d/16cLV99FBR3Wymv74Aaj0iIVeY_HOyYUE3YUOg7kew1I/edit?resourcekey=0-2vih9Q9RWQgVjg6I5LP-zg